imagetool/src/main/resources/docker-files/Create_Image.mustache

```diff
@@ -187,12 +188,6 @@ ENV ORACLE_HOME={{{oracle_home}}} \
     PROPERTIES_FILE_DIR={{{oracle_home}}}/properties \
     PATH=${PATH}:{{{java_home}}}/bin:{{{oracle_home}}}/oracle_common/common/bin:{{{oracle_home}}}/wlserver/common/bin:{{{oracle_home}}}{{#isWdtEnabled}}:${{{domain_home}}}/bin{{/isWdtEnabled}}
 
-## Create user and group
```

Comment:
This change seems unrelated to tar/gzip.

Reply:
If you mean the create user and group, then yes, its related.  The OS_UPDATE intermediate layer that was originally thrown away already has the user and group created.  So, there is no reason to create them again.